binr/r2pm/r2pm

```diff
@@ -327,16 +330,24 @@ R2PM_GIT() {
 	cd "${R2PM_GITDIR}"
 	if [ -d "${DIR}" ]; then
 		cd "${DIR}"
-		git checkout master
-		git pull
+		if [ "${R2PM_GITMASTER}" = "master" ]; then
```

Comment:
uhm R2PM vars are suposed to be set from the packages, not from the user who is calling it. but still.. i want to understand better the problem you are trying to solve here:

- master will always be master, better call it R2PM_GITBRANCH
- this var must be set by the package itself (do u have an example of it?)

Reply:
I just want to call `r2pm -i atombios` from a CI without having to hard code a branch name into the package itself.
Therefore, with this change as is, it lets you export R2PM_GITMASTER=<sha1> and it wont try to switch away to master or git pull, it will just check out the commit specified in the env var.  This is convenient because you can get the git sha1 from the CI environment action variables and export it as a bash variable.

Reply:
Example here https://github.com/zamaudio/radare2-extras/commit/8836682eb9d2ea4ae5e8d10114ed8920f549438d#diff-e9f950f17198d3d5e3122a44230a09b9R38-R42

Reply:
Without this functionality, you cannot test PRs on radare2-extras.